packages/google/src/google-generative-ai-language-model.ts

```diff
@@ -109,6 +109,7 @@ export class GoogleGenerativeAILanguageModel implements LanguageModelV1 {
         this.supportsStructuredOutputs
           ? convertJSONSchemaToOpenAPISchema(responseFormat.schema)
           : undefined,
+      audioTimestamp: this.settings.audioTimestamp,
```

Comment:
Most of the time `this.settings.audioTimestamp` won't be defined, but as written this will then add `audioTimestamp: undefined` to every request.

Can we alter this to something like the below to avoid that?

`...(this.settings.audioTimestamp && { audioTimestamp: this.settings.audioTimestamp }`